lib/tiki/torch/consumers/publisher.rb

```diff
@@ -3,10 +3,14 @@ module Torch
     class Consumer
       module Publishing
 
-        def publish(queue_name, payload, properties = {})
-          defaults = Torch.config.default_message_properties
-          custom   = { parent_message_id: event.message_id }.merge properties.dup
-          Torch.publish queue_name, payload, defaults.merge(custom)
+        def publish(queue_name, payload, properties = {}, format = config.transcoder_code )
```

Comment:
`config.transcoder_code` assumes that this module will be included in a class that already has a `#config` method. Maybe the default for `format` should be `Torch.config.transcoder_code`?